### DISCUSSION TITLE
Is there any way to use the product of the mc category?

### DISCUSSION THREAD

**Discussion by adppark** [tuya/tuya-homebridge#240] [Q&A]
I am using the MC category product which is door and window controller to control the window.
The switch is connected to the motor and operates as an open, close stop button and automatically stops when the motor cannot proceed any further.
I want to use this product in homekit through homebridge...

Is there any plan to support mc category in tuya-home bridge?
or.. Is there any way to re-use the previously developed plug-in? (Forexample, add to case 'mc' statement in index.js?)

**Comment by 0x5e** [comment]
> I am using the MC category product which is door and window controller to control the window.
> 
> The switch is connected to the motor and operates as an open, close stop button and automatically stops when the motor cannot proceed any further.
> 
> I want to use this product in homekit through homebridge...
> 
> 
> 
> Is there any plan to support mc category in tuya-home bridge?
> 
> or.. Is there any way to re-use the previously developed plug-in? (Forexample, add to case 'mc' statement in index.js?)

try my fork repo
